app/backend/server/api/v1/users_router.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+from datetime import date
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Query, Depends, Security
+
+from server import auth
+
+from db import crud
+from db import schemas
+from db.database import get_db
+
+router = APIRouter()
+
+
+@router.get("/{user_id}", response_model=schemas.User)
+async def get_user(
+    user_id: int,
+    db: Session = Depends(get_db),
+    current_user: schemas.User = Security(auth.get_current_user,
+                                          scopes=["personal"])
+):
+    return crud.get_user(
+            db=db, 
+            user_id=user_id
+        )
+
+
+@router.delete("/{user_id}")
+async def delete_user(
+    user_id: int = None,
+    db: Session = Depends(get_db),
+    current_user: schemas.User = Security(auth.get_current_user,
+                                          scopes=["personal"])
+):
+    crud.delete_user(db=db, user_id=user_id)
+
+
+@router.get("/{user_id}/commander_id")
+async def get_commander(
+    user_id: int = None,
+    db: Session = Depends(get_db),
+    current_user: schemas.User = Security(auth.get_current_user,
+                                          scopes=["personal"])
+) -> int:
+    return crud.get_commander_id(db=db, user_id=user_id)
+
+
+@router.get("/me", response_model=schemas.User)
+async def get_current_user(
+    current_user: schemas.User = Security(auth.get_current_user,
+                                          scopes=["personal"])
+):
+    return current_user
+
+
+@router.get("/subjects", response_model=List[schemas.User])
```

Comment:
Doesn't even make sense..
It should be under /users/{I'd}/subjects

Reply:
why should i be able to access anyone elses subjects?

Reply:
ive changed it. w/e